semver/parse_range_test.ts

```diff
@@ -666,3 +666,24 @@ Deno.test("parseRange() throws on invalid range", () => {
     'Cannot parse version range: range "blerg" is invalid',
   );
 });
+
+Deno.test("parseRange() handles wildcards", () => {
+  assertEquals(parseRange("<1.*"), [
+    [{ operator: "<", major: 1, minor: 0, patch: 0 }],
+  ]);
+  assertEquals(parseRange("<1.*.0"), [
+    [{ operator: "<", major: 1, minor: NaN, patch: 0 }],
+  ]);
```

Comment:
The difference of these 2 results seem strange to me. Is this intentionally in this way?

Reply:
This seems indeed wrong. Imo `<1.*.0` should be an invalid range and throw.

Reply:
I took a closer look at this, `satisfies()` tests allow this: 
https://github.com/denoland/std/blob/3b75ee7c1925388e01dd69540d51d9387e26dcb0/semver/satisfies_test.ts#L75C5-L75C25

Reply:
I think the range expression `<1.*.0` is fine, but shouldn't the parse result be the same as `<1.x` as they seem expressing the same range?